### PR TITLE
cronspec details

### PR DESCRIPTION
**Pull request by lbliii**
- restructured content
- added full list of "nonstandard" cron inputs (eg @hourly)
- added callouts
- added SQL ingest + jsonnet example of cronspec config

### CODE REVIEW

File: doc/docs/2.3.x/concepts/pipeline-concepts/pipeline/cron.md [hunk 1]
```diff
@@ -1,69 +1,79 @@
...
-    ```json
-      "input": {
-        "cron": {
-          "name": "tick",
-          "spec": "@every 10s",
-          "overwrite": true
-        }
-      }
-    ```
+| `"spec"`   | The interval between scheduled cron jobs; accepts [RFC 3339](https://www.ietf.org/rfc/rfc3339.txt) inputs as well as `never`, `@hourly`, `@daily`, `@weekly`, `@monthly`, `@yearly`, `@reboot`, and `@every ...`|
```
Comment (BOsterbuhr) [inline doc/docs/2.3.x/concepts/pipeline-concepts/pipeline/cron.md:30]: What does `@reboot` do? The others are more obvious, but maybe we should add a little note on `@reboot` and also what can be used with `@every`.
For example, if I want it to be every 3 months, what would that be? `@every 3m` would be minutes, not months, right?

**Review by BOsterbuhr — CHANGES_REQUESTED** [review]
Good work; things are clean and concise. However, I have one request regarding what can be used for `spec`.

File: doc/docs/2.3.x/concepts/pipeline-concepts/pipeline/cron.md (hunk 1 above)
Comment (lbliii) [reply doc/docs/2.3.x/concepts/pipeline-concepts/pipeline/cron.md]: Found some direct links to those supported formats in the cron.go pkg!

**Review by BOsterbuhr — APPROVED** [review]
LGTM

**Review by maguenza — APPROVED** [review]
LGTM - will share on the FREQQ ticket as well.

**Pull request merged**